i18n/fr.jsonp.js

```diff
@@ -19,8 +19,14 @@ document.localeJson = {
       "configure-title": "Configuration",
       "configure-about-usage-link": "Informations (Utilisation)",
       "configure-btn-library": "Bibliothèque ZIM",
+      "configure-btn-folderselect": "Sélectionner un dossier",
+      "configure-btn-rescan": "Rechercher",
+      "configure-about-rescan-tip": "Réanalyser la carte SD et la mérmoire interne",
+      "configure-select-file-numbers": "archive(s) trouvées dans le lieu sélectionné. ",
```

Comment:
You also need to put parentheses round the plural form of the adjective, i.e. "trouvée(s)"